src/Cake.Common/Tools/NuGet/Pack/NuGetPacker.cs

```diff
@@ -214,6 +214,12 @@ private ProcessArgumentBuilder GetArguments(FilePath filePath, NuGetPackSettings
                     settings.Properties.Select(property => string.Concat(property.Key, "=", property.Value))));
             }
 
+            // Keep temporarily autogenerated NuSpec file?
+            if (settings.KeepTemporaryNuSpecFile)
+            {
+                builder.Append("-KeepTemporaryNuSpecFile");
```

Comment:
Ah, I see. I understand why I lost you. I'm obviously in another planetary system. I'll get back to earth and remove this right away. :smile: